build.sh

```diff
@@ -5,7 +5,7 @@ local_bin="$HOME/.local/bin/"
 local_include="$HOME/.local/include/csq"
 csq_bin="$local_bin/csq"
 csq_path="$local_include"
-shell_config="$HOME/.${SHELL##*/}rc"
+shell_config="$HOME/.bashrc"
```

Comment:
Using this approach assumes that the user is using the Bash shell and may not work correctly if the user is using a different shell.